### ISSUE TITLE
Sidebar collapsed by default

### ISSUE THREAD

**Issue by gvreddy04** [vikramlearning/blazorbootstrap#700]
### Discussed in https://github.com/vikramlearning/blazorbootstrap/discussions/699

<div type='discussions-op-text'>

<sup>Originally posted by **heather-timmins** April 30, 2024</sup>
Hi, 

Sorry if i've missed something but can you make the sidebar collapsed by default? and only toggle to open? 

Thanks

![image](https://github.com/vikramlearning/blazorbootstrap/assets/144691290/5a9d0420-e25e-4979-8364-2f67c78792bc)
</div>

**Comment by psijkof** [comment]
Yes it would be great to have an option InitiallyCollapsed oslt. Now I invoke _sidebar.ToggleSidebar() on initialization. (OnAfterRender)...

**Comment by GaryWayneSmith** [comment]
It seems this is the default in the current version and I need to go just the opposite direction.  My desktop implementations are all defaulting to hide.  I don't have an issue with the default the way it is, it would be nice to have a parameter to specify the initial value.  Toggle assumes that we know the initial state.  If you change it to default, all implementations using toggle on initialize will have to be updated.

**Comment by LakinduK** [comment]
This would be a great to have feature. Any update on this?
